.devcontainer/Dockerfile

```diff
@@ -0,0 +1,39 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+FROM mcr.microsoft.com/dotnet/core/sdk:2.1.602
+
+# Avoid warnings by switching to noninteractive
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Configure apt and install packages
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends apt-utils 2>&1 \
+    #
+    # Verify git, process tools, lsb-release (common in install instructions for CLIs) installed
+    && apt-get -y install git procps lsb-release \
+    #
+    # Clean up
+    && apt-get autoremove -y \
+    && apt-get clean -y \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install PowerShell
+RUN apt-get update \
+    && apt-get install curl gnupg apt-transport-https -y \
+    && curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - \
+    && echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-debian-stretch-prod stretch main" > /etc/apt/sources.list.d/microsoft.list \
+    && apt-get update \
+    && apt-get install -y powershell
+
+# Switch back to dialog for any ad-hoc use of apt-get
+ENV DEBIAN_FRONTEND=dialog
+
+SHELL ["/usr/bin/pwsh", "-Command"]
+
+# Install PowerShell modules required for building and testing
+RUN Install-Module InvokeBuild -Force; Install-Module Pester -Force
+RUN Install-Module platyPS -Force; Install-Module Pester -Force
+RUN Install-Module Pester -Force; Install-Module Pester -Force
```

Comment:
can you simplify this to one `Install-Module`? I don’t think we need to install Pester 4 times 😅

Reply:
Sorry, that's definitely a copy-paste error. If PowerShellGet was sensible, it would perform a check before actually downloading and installing the same thing again....